### PR TITLE
V3 Balance Sheet

### PR DESCRIPTION
**Pull request by offerijns**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@centrifuge/tinlake-js",
+  "name": "@centrifuge/tinlake-js-v3",
```
Comment (philipstanislaus) [inline package.json:2]: Why this change?

File: src/actions/analytics.ts [hunk 2]
```diff
@@ -6,6 +6,10 @@ import BN from 'bn.js'
 export function AnalyticsActions<ActionsBase extends Constructor<TinlakeParams>>(Base: ActionsBase) {
   return class extends Base implements IAnalyticsActions {
     // borrower analytics
+    getNetAssetValue = async (): Promise<BN> => {
```
Comment (philipstanislaus) [inline src/actions/analytics.ts:9]:
```suggestion
    getCurrentNAV = async (): Promise<BN> => {
```

**Review by philipstanislaus — APPROVED** [review]
🎉

File: package.json (hunk 1 above)
Comment (offerijns) [reply package.json:2]: Ah I need to revert it, that's needed for making `npm link` work locally..
